src/scripts/components/stories/story/story.styl

```diff
@@ -13,3 +13,19 @@
 
   > *
     width: 50%
+
+.rightSideComponent
+  height: 100%
+
+@media screen and (max-width: 480px)
+  .main
+    display: flex
+    flex-direction: column
+    overflow-y: auto
+
+    > :last-child
```

Comment:
do we have a className for this element?

Reply:
Sadly no, it can be either the gallery, story video or the globe.

Reply:
ah ok, then 👍